app-crypt/ssdeep/ssdeep-2.14.1-r1.ebuild

```diff
@@ -12,33 +12,21 @@ SRC_URI="https://github.com/${PN}-project/${PN}/releases/download/release-${PV}/
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv x86"
-IUSE="static-libs"
-
-DOCS=(
-	AUTHORS
-	ChangeLog
-	FILEFORMAT
-	NEWS
-	README
-	TODO
-)
 
 PATCHES=(
 	"${FILESDIR}/${PN}-2.10-shared.patch"
 )
 
 src_prepare() {
 	default
+
```

Comment:
I don't see the benefit of adding an empty line to two-line function, tbh, and it only makes the ebuild harder to fit on screen.

Reply:
Sure.